appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/EJBContainerTransactionManager.java

```diff
@@ -96,12 +90,8 @@
         container = (BaseContainer)c;
         ejbDescriptor = ejbDesc;
         transactionManager = ejbContainerUtilImpl.getTransactionManager();
-
-        IASEjbExtraDescriptors iased = ejbDesc.getIASEjbExtraDescriptors();
-
-        if(iased.getCmtTimeoutInSeconds() != 0){
```

Comment:
Original code: set it to 0 and if something change the zero to another number.
Now: if something, set it to a value and to zero otherwise. And it is final.